src/content/docs/en/reference/adapter-reference.mdx

```diff
@@ -50,26 +50,30 @@ interface AstroAdapter {
 	exports?: string[];
 	args?: any;
 	adapterFeatures?: AstroAdapterFeatures;
-	supportedAstroFeatures?: AstroFeatureMap;
+	supportedAstroFeatures: AstroAdapterFeatureMap;
 }
 
 export interface AstroAdapterFeatures {
 	/**
 	 * Creates an edge function that will communicate with the Astro middleware.
 	 */
 	edgeMiddleware: boolean;
+	/**
+	 * Determine the type of build output the adapter is intended for. Defaults to `server`;
+	 */
+	buildOutput?: 'static' | 'server';
 }
 
-export type SupportsKind = 'unsupported' | 'stable' | 'experimental' | 'deprecated';
+export type AdapterSupportsKind = 'unsupported' | 'stable' | 'experimental' | 'deprecated' | 'limited';
 
 export type AdapterSupportWithMessage = {
-	support: Exclude<SupportsKind, 'stable'>;
+	support: Exclude<AdapterSupportsKind, 'stable'>;
 	message: string;
 };
 
-export type AdapterSupport = SupportsKind | AdapterSupportWithMessage;
+export type AdapterSupport = AdapterSupportsKind | AdapterSupportWithMessage;
 
-export type AstroFeatureMap = {
+export type AstroAdapterFeatureMap = {
```

Comment:
This [was renamed](https://github.com/withastro/astro/blob/948ad8eecf71f1f6de33855a6065925d7858d28d/packages/astro/src/types/public/integrations.ts#L100).